### PR TITLE
[vs tests]: Add mirror session warm reboot test

### PR DESCRIPTION
**Pull request by daall**
**What I did**
Added a test case to validate that the monitor port is preserved during warm reboot.

**Why I did it**
To validate #1054 

**How I verified it**
```
daall@valhalla:~/dev/sonic-swss/tests$ sudo pytest -s -v --dvsname=vs test_warm_reboot.py::TestWarmReboot::test_MirrorSessionWarmReboot -s
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15+, pytest-3.3.0, py-1.8.0, pluggy-0.6.0 -- /usr/bin/python2
cachedir: .cache
rootdir: /home/daall/dev/sonic-swss/tests, inifile:
collected 1 item                                                                                                                                                                                                  

test_warm_reboot.py::TestWarmReboot::test_MirrorSessionWarmReboot remove extra link dummy
PASSED                                                                                                                                    [100%]

============================================================================================ 1 passed in 78.33 seconds ============================================================================================
```

**Details if related**
Continuation of #1097

### CODE REVIEW

**Comment by daall** [comment]
retest this please

**Comment by daall** [comment]
retest this please

**Comment by daall** [comment]
retest this please

**Comment by daall** [comment]
Reopening this PR because the feature branch was deleted inadvertently. 🤦‍♂

File: tests/test_warm_reboot.py [hunk 1]
```diff
@@ -234,7 +234,121 @@ def ping_new_ips(dvs):
...
 class TestWarmReboot(object):
+    def test_MirrorSessionWarmReboot(self, dvs, testlog):
+        """
+            This test verifies that mirror session monitor ports are not
+            changed after warm reboot.
+        """
+
+        # Setup the virtual switch
+        dvs.setup_db()
+        dvs.runcmd("config warm_restart enable swss")
```
Comment (yxieca) [inline tests/test_warm_reboot.py]: I suggest moving warm restart setting close to swss restart.

File: tests/test_warm_reboot.py [hunk 2]
```diff
@@ -234,7 +234,121 @@ def ping_new_ips(dvs):
...
+        create_mirror_session(dvs, "test_session3", "1.1.1.1", "2.2.2.2", "0x6558", "8", "100", "0")
+
+        test_session1_mp = get_mirror_session_state(dvs, "test_session1")["monitor_port"]
+        test_session2_mp = get_mirror_session_state(dvs, "test_session2")["monitor_port"]
+        test_session3_mp = get_mirror_session_state(dvs, "test_session3")["monitor_port"]
+
+        # Perform the warm reboot
+        dvs.stop_swss()
+        dvs.start_swss()
+        time.sleep(5)
```
Comment (yxieca) [inline tests/test_warm_reboot.py]: is sleep 5 seconds enough? You could polling "show warm_restart state" and wait until orchagent is reconciled. That would take lot longer than 5 seconds though.

**Pull request closed without merging**